index.html

```diff
@@ -431,12 +449,71 @@
 application caches</a> or from local resources.</p>
 </dd>
 
+<dt>readonly attribute unsigned short transferSize</dt>

```

Comment:
Any reason why *Size attributes are commented out? We define them in RT, is the intent to drop this text from NT?